deployment/terraform/assets/cluster.tf

```diff
@@ -592,3 +592,53 @@ resource "aws_security_group" "proxy" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 }
+
+resource "aws_instance" "job_server" {
+  tags = {
+    Name = "${var.cluster_name}-job-server-${count.index}"
+  }
+
+  connection {
+    # The default username for our AMI
+    type = "ssh"
+    user = "ubuntu"
+    host = self.public_ip
+  }
+
+  ami           = "ami-0ac80df6eff0e70b5" # 18.04 LTS
+  instance_type = var.job_server_instance_type
+  key_name      = aws_key_pair.key.id
+  count         = var.job_server_instance_count
+  vpc_security_group_ids = [
+    aws_security_group.app[0].id,
+  ]
+
+  dynamic "root_block_device" {
+    for_each = var.root_block_device
+    content {
+      volume_size = lookup(root_block_device.value, "volume_size", null)
+      volume_type = lookup(root_block_device.value, "volume_type", null)
+    }
+  }
+
+  provisioner "file" {
+    source      = var.mattermost_license_file
+    destination = "/home/ubuntu/mattermost.mattermost-license"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "while [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; sleep 1; done",
+      "wget --no-check-certificate -qO - https://s3-eu-west-1.amazonaws.com/deb.robustperception.io/41EFC99D.gpg | sudo apt-key add -",
+      "wget --no-check-certificate -qO - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -",
+      "sudo sh -c 'echo \"deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main\" > /etc/apt/sources.list.d/pgdg.list'",
+      "sudo apt-get -y update",
+      "sudo apt-get install -y mysql-client-5.7",
```

Comment:
Curious why we need this?

Reply:
Discussed offline, generally speaking we won't need these but still it's good to have same tools as app server. Also, in such scenarios like debugging the job server these might come handy.

Reply:
I've actually added this which is needed to load sql dumps during automated comparisons and also do other things like db reset without using the CLI.